File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/InvoiceUnderAssessReason.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class InvoiceUnderAssessReason : AuditDetails
    {
        public int? InvoiceId { get; set; }
        public int? TebaInvoiceId { get; set; }
        public int UnderAssessReasonId { get; set; }
        public string UnderAssessReason { get; set; }
        public string Comments { get; set; }
        public InvoiceStatusEnum InvoiceStatus { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/invoice-under-assess-reason.ts
import { InvoiceStatusEnum } from "../enums/invoice-status.enum";

export class InvoiceUnderAssessReason {
    invoiceId: number| null;
    tebaInvoiceId: number| null;
    underAssessReasonId: number;
    underAssessReason: string;
    comments: string;
    invoiceStatus: InvoiceStatusEnum;
}
